tests/test_smt.py

```diff
@@ -34,25 +34,29 @@
     <smtInfo fingerprint="00:11:22:33"
      SMTserverIP="192.168.1.1"
      SMTserverName="fantasy.example.com"
+     SMTregistryName="registry-fantasy.example.com"
      region="antarctica-1"/>''')
 
 smt_data_ipv6 = dedent('''\
     <smtInfo fingerprint="00:44:22:33"
      SMTserverIPv6="fc00::1"
      SMTserverName="fantasy.example.com"
+     SMTregistryName="registry-fantasy.example.com"
      region="antarctica-1"/>''')
 
 smt_data_ipv46 = dedent('''\
     <smtInfo fingerprint="00:11:22:33"
      SMTserverIP="192.168.1.1"
      SMTserverIPv6="fc00::1"
      SMTserverName="fantasy.example.com"
+     SMTregistryName="registry-fantasy.example.com"
      region="antarctica-1"/>''')
 
 smt_data_no_region = dedent('''\
     <smtInfo fingerprint="00:11:22:33"
      SMTserverIP="192.168.1.1"
-     SMTserverName="fantasy.example.com"/>''')
+     SMTserverName="fantasy.example.com"
+     SMTregistryName="registry-fantasy.example.com"/>''')
```

Comment:
There should be a test setup without `SMTregistryName`

Reply:
You mean empty value or no entry at all ?
Because currently the presence of the entry is mandatory [here](https://github.com/SUSE-Enceladus/cloud-regionsrv-client/pull/157/files#diff-9811adbce77061bf864555ec5e53fec8f5e8b2e87fa91b1618dabe3ef564a139R43)
the registry entry presence on the RMT server (smt info) should be mandatory, shouldn't it ?

Reply:
I will change it to non mandatory as maybe 3rd parties users have an RMT version excluding or not using the registry at all